script/Arcueid/Day 10/10_00_ARC10_8_2.txt

```diff
@@ -107,7 +107,8 @@ This is not something that can be forgiven with a mere 'I'm sorry'.
 [sha:a6ecfe79db3bdfb8e6c5ad59e93724354872683c]{
 -- Page 870, Offset 26791.
 -- 「気にしないで。……謝らなくちゃいけないのは、わたしの方なんだから」
-"Don't worry about it... I'm the one who should be apologizing"
+// This should have a period despite the JP text, I think
```

Comment:
Yeah I think speech should be properly punctuated in EN even when it isn't in the JP. Just easy to miss.